cloud_edition/flexibility_mode/docs/crontasks.rst

```diff
@@ -46,25 +51,27 @@ If you don't want to use this wrapper you can prepend `SHELL=/bin/bash`, for exa
 
 .. code-block:: bash
 
-    SHELL=“/usr/local/sbin/cron_wrapper.sh”
+    SHELL="/usr/local/sbin/cron_wrapper.sh"
+
+    # email adress below is only used to notify if anything goes wrong. Feel free to adapt it to your needs!
     MAILTO="projectmanager@acme.com"
 
     #Ansible: akeneo:rule:run
-    15 12,20 * * * akeneo:rule:run --env=prod
```

Comment:
Set the --env=prod for PIM older than 4.x

Reply:
Then it should be on older branches not 4.x